packages/types/README.md

```diff
@@ -438,3 +438,21 @@ sdk.build([
   ])
 ])
 ```
+
+### Path
+
+```javascript
+import * as t from "@onflow/types"
+
+sdk.build([
+  sdk.args([
+    sdk.arg(
+      {
+        domain: "public"                // public | private | storage
```

Comment:
Can we also update the Quick Reference with this new type?

Reply:
Will do in the subsequent PR to add this update to FCL